cmd/jaas/cmd/upgradeto.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2025 Canonical.
+
+package cmd
+
+import (
+	"github.com/juju/cmd/v3"
+	"github.com/juju/gnuflag"
+	jujuapi "github.com/juju/juju/api"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/jujuclient"
+
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/pkg/api"
+	apiparams "github.com/canonical/jimm/v3/pkg/api/params"
+)
+
+const (
+	upgradeToDoc = `
+Upgrades a controller to a specified version.
+`
+	upgradeToExample = `
+    juju upgrade-to 2cb433a6-04eb-4ec4-9567-90426d20a004 --version 2.0.1
+    juju upgrade-to mymodel --version 3.5.0
```

Comment:
Is this valid? Just the `myModel` part I mean. I think it needs an `owner/` portion. But then the PR description says we won't do names and UUIDs.